topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml

```
@@ -35,6 +35,31 @@ Resources:
           hidden: false
     Tags:
       - CC*
+  WSU-GRID-CE1:
+    Active: true
+    ID: 1099
```

```suggestion
    ID: 1352
```

@jdost321 in the future, you can just push another commit on the relevant branch to update the PR